src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateCapacity", "HashMap.Test\HashMapNamespace.CalculateCapacity\HashMapNamespace.CalculateCapacity.csproj", "{52CF5877-0FA0-482F-A9E8-137C1FF052EC}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ProvideCapacity", "HashMap.Test\HashMapNamespace.ProvideCapacity\HashMapNamespace.ProvideCapacity.csproj", "{6B3046FE-22DE-43B3-9335-83883D159BA1}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Add", "HashMap.Test\HashMapNamespace.Add\HashMapNamespace.Add.csproj", "{8513B597-4E9C-44CD-809F-290F446E21E4}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.AddOrUpdate", "HashMap.Test\HashMapNamespace.AddOrUpdate\HashMapNamespace.AddOrUpdate.csproj", "{DD451542-3BE6-47B2-8D29-B614A0EE2F6B}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Remove", "HashMap.Test\HashMapNamespace.Remove\HashMapNamespace.Remove.csproj", "{2FB8A7E4-D031-466B-8834-DA4DA8265808}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ContainsKey", "HashMap.Test\HashMapNamespace.ContainsKey\HashMapNamespace.ContainsKey.csproj", "{49821606-F17E-4E4A-A1D0-5F5D052120F9}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.TryGetValue", "HashMap.Test\HashMapNamespace.TryGetValue\HashMapNamespace.TryGetValue.csproj", "{E252454D-7F25-45DC-9CF1-C6DB354EB67D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Clear", "HashMap.Test\HashMapNamespace.Clear\HashMapNamespace.Clear.csproj", "{C54A2E51-5AAA-42BE-BF3D-C5BF7FD9C557}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.GetEnumerator", "HashMap.Test\HashMapNamespace.GetEnumerator\HashMapNamespace.GetEnumerator.csproj", "{CB575984-6982-4487-8944-B4751E0970F0}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.HashMap64Node", "HashMap.Test\HashMapNamespace.HashMap64Node\HashMapNamespace.HashMap64Node.csproj", "{1ABBB916-2C9C-4558-95EE-EDDCEBB4A8A1}"
```

Comment:
### 5/5 Tests Pass.
```
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj (in 433 ms).
  1 of 2 projects are up-to-date for restore.
  HashMap -> /workspaces/CodeExamples/src/HashMap/bin/Debug/net6.0/HashMap.dll
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(89,20): warning CS8618: Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable. [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(158,21): warning CS8625: Cannot convert null literal to non-nullable reference type. [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(125,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(133,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(141,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMap64Node.cs(152,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/HashMapNamespace.HashMap64Node.csproj]
  HashMapNamespace.HashMap64Node -> /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/bin/Debug/net7.0/HashMapNamespace.HashMap64Node.dll
Test run for /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node/bin/Debug/net7.0/HashMapNamespace.HashMap64Node.dll (.NETCoreApp,Version=v7.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.2 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.

Passed!  - Failed:     0, Passed:     5, Skipped:     0, Total:     5, Duration: 39 ms - HashMapNamespace.HashMap64Node.dll (net7.0)
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.HashMap64Node (roost-1709036480) $ 
```